<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btncadastroclientemenu_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 169, 39);</string>
    </property>
    <property name="text">
     <string>Cadastrar / Atualizar Cliente</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Inserir_um_t√≠tulo-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnconsultarclientemenu_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 162, 37);</string>
    </property>
    <property name="text">
     <string>Consultar Cliente</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblfundo_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>921</width>
      <height>921</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 214, 214);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnvoltarmenuprincipal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(165, 165, 165);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>botao-voltar.png</normaloff>botao-voltar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <zorder>lblfundo_2</zorder>
   <zorder>btncadastroclientemenu_2</zorder>
   <zorder>lblLogo_2</zorder>
   <zorder>btnconsultarclientemenu_2</zorder>
   <zorder>btnvoltarmenuprincipal</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
